Sidney-/slugtrade
# Bugfix for Issue #152 (Port number of start script is always 8000.)

Issue:
- The port number can't be specified on the command line when running the start script.

How to test:
- Run "start.sh" and verify that the app is running on localhost:8000
- Run "start.sh 8001" and verify that the app is running on localhost:8001
- Run "first_start.sh" and verify that the app is running on localhost:8000
- Run "first_start.sh 8001" and verify that the app is running on localhost:8001

Note: the Windows scripts still only use the default port number.
